### PR TITLE
Adding LLAP Interpreter to Hue.ini and PseudoDistributed.ini.tmpl

### PR DESCRIPTION
**Pull request by alericmckearn**
Will follow with full setup guide.

### CODE REVIEW

**Comment by alericmckearn** [comment]
Notes:
--removed settings frim inis based on update to locmemcache
--fixed static llap thrift port
--added documentation to administration manual
--added llap interprester to inis

File: docs/admin-manual/manual.md [hunk 1]
```diff
@@ -263,6 +263,43 @@ The download feature in the file browser can be disabled separately with the fol
       # Port where HiveServer2 Thrift server runs on.
       ## hive_server_port=10000
 
+### LLAP
```
Comment (romainr) [inline docs/admin-manual/manual.md]: Note: this doc content should probably moved to https://github.com/cloudera/hue/blob/master/docs/docs-site/content/administrator/configuration/editor/_index.md#hive (as the real doc website is now there)

**Comment by romainr** [comment]
https://issues.cloudera.org/browse/HUE-8887

**Comment by romainr** [comment]
https://review.cloudera.org/r/14066/

File: docs/admin-manual/manual.md (hunk 1 above)
Comment (romainr) [reply docs/admin-manual/manual.md]: https://review.cloudera.org/r/14066/

**Pull request merged**